h2o-core/src/main/java/hex/ScoringInfo.java

```diff
@@ -299,7 +299,6 @@ public static TwoDimTable createScoringHistoryTable(ScoringInfo[] scoringInfos,
           table.set(row, col++, si.scored_xval != null ? si.scored_xval._r2 : Double.NaN);
         }
         if (isClassifier) {
-          table.set(row, col++, si.scored_xval != null ? si.scored_xval._rmse : Double.NaN);
           table.set(row, col++, si.scored_xval != null ? si.scored_xval._logloss : Double.NaN);
```

Comment:
There are three kinds of _rmse, one belongs to train, one belongs to validation and one belongs to xval.  Not sure where the duplicated _rmse is....

Reply:
Yeah, that one was well hidden before my eyes :)